src/Form/SettingsForm.php

```diff
@@ -0,0 +1,160 @@
+<?php
```

Comment:
we should add a schema.yml for the settings: https://www.drupal.org/docs/8/api/configuration-api/configuration-schemametadata